backend/api/views/compute_plan_graph.py

```diff
@@ -19,19 +19,25 @@
 logger = structlog.get_logger(__name__)
 
 TASK_CATEGORY_INPUTS = {
-    ComputeTask.Category.TASK_TRAIN: ["in/model"],
-    ComputeTask.Category.TASK_TEST: ["in/predictions"],
-    ComputeTask.Category.TASK_COMPOSITE: ["in/head_model", "in/trunk_model"],
-    ComputeTask.Category.TASK_AGGREGATE: ["in/models[]"],
-    ComputeTask.Category.TASK_PREDICT: ["in/tested_model"],
+    ComputeTask.Category.TASK_TRAIN: [{"id": "in/model", "kind": "model"}],
+    ComputeTask.Category.TASK_TEST: [{"id": "in/predictions", "kind": "model"}],
+    ComputeTask.Category.TASK_COMPOSITE: [
+        {"id": "in/head_model", "kind": "model"},
+        {"id": "in/trunk_model", "kind": "model"},
+    ],
+    ComputeTask.Category.TASK_AGGREGATE: [{"id": "in/models[]", "kind": "model"}],
+    ComputeTask.Category.TASK_PREDICT: [{"id": "in/tested_model", "kind": "model"}],
```

Comment:
Why do we still infer inputs / outputs from categories?

Reply:
There is a follow up addressing exactly that: https://github.com/Substra/substra-backend/pull/486

Reply:
Thanks